spec/models/game_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe Game do
+  it { should validate_presence_of(:start_time) }
```

Comment:
A lot of times I find it nice to break up association and validation tests with a describe:

```ruby
describe Game do
  describe 'Associations' do
    # ...
  end

  describe 'Validations' do
    # ...
  end
end
```